OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -934,7 +934,144 @@ internal static void UpgradeActors(int engineVersion, ref List<MiniYamlNode> nod
 		{
 			foreach (var node in nodes)
 			{
-				// Add rules here
+				// Fix RA building footprints to not use _ when it's not necessary
+				if (engineVersion < 20160530 && Game.ModData.Manifest.Mod.Id == "ra")
```

Comment:
Can you please pass `ModData modData` in as the first argument to these methods?  Referencing `Game.ModData` causes a bunch of issues, and i'm slowly trying to eliminate all these references.